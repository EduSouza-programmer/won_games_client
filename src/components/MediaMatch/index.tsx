import * as S from "./MediaMatch.styles";

function MediaMatch() {
  return (
    <S.Wrapper>
      <h1>MediaMatch</h1>
    </S.Wrapper>
  );
}

export default MediaMatch;
