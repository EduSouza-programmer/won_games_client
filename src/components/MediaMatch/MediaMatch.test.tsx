import { screen } from "@testing-library/react";
import { renderWithTheme } from "@/utils/tests/helpers";

import MediaMatch from ".";

describe("<MediaMatch />", () => {
  it("should render the heading", () => {
    const { container } = renderWithTheme(<MediaMatch />);

    expect(
      screen.getByRole("heading", { name: /MediaMatch/i })
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
