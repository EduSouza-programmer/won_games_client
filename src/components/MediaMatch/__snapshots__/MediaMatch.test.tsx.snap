// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MediaMatch /> should render the heading 1`] = `
.c0 {
  background-color: #F231A5;
}

<div
  class="c0"
>
  <h1>
    MediaMatch
  </h1>
</div>
`;
